### DISCUSSION TITLE
Lesson 1: Unable to locate this TxnHash

### DISCUSSION THREAD

**Discussion by RomeoChen** [Cyfrin/foundry-full-course-cu#1129] [Q&A]
When I chose alchemy to send me 0.5 Sepolia ETH, after I click send button, there is a transaction generated blow:
<img width="852" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/30121702/747a37bc-c1ba-41c8-91b5-42f01e43bf2e">

But when I click it, something goes wrong
<img width="1337" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/30121702/50a8bb74-8f8b-473c-91a9-99ac2e95bd5d">

I'm new here. Could anyone help me solve this problem? or let me know what should I do.

**Comment by theirrationalone** [comment]
Alternatively you can go to your metamask activity scroll and can check that tx if it really happened....!
